packages/react/src/elements/components/Banner/types.js

```diff
@@ -0,0 +1,7 @@
+/** @todo Move into shared package to consolidate component types */
+export default Object.freeze({
+  PRIMARY: "primary",
+  SUCCESS: "success",
+  DANGER: "danger",
```

Comment:
I got some clarification from the designers that their preferred word for this state is `error` rather than `danger`.